test/cmdlineTests.sh

```diff
@@ -156,6 +156,9 @@ function test_solc_behaviour()
     else
         sed -i -e '/^Warning: This is a pre-release compiler version, please do not use it in production./d' "$stderr_path"
```

Comment:
This would be way nicer:
```suggestion
        sed -i -e '/^Warning: This is a pre-release compiler version, please do not use it in production./,+1d' "$stderr_path"
```
But OS X sed doesn't support it, unfortunately...

Reply:
What about the issue that `-i` requires an argument on mac os?

Reply:
Hm... there wasn't an argument to it before and still isn't, but it seems to work either way... looking at the man page and e.g. https://stackoverflow.com/a/4247319 this would supposedly use ``-e`` as the argument for ``-i``, but that should actually not work... I'm still logged into a CircleCI macos SSH session - I'll investigate this...

Reply:
Hm... at least in the CircleCI MacOS environment it looks like ``sed -i -e`` works and behaves just like GNU sed...
Maybe we should use ``-i.bak`` everywhere anyways, though...

Reply:
I changed it everywhere in cmdlineTests.sh now.